src/components/Select/index.tsx

```diff
@@ -1,27 +1,53 @@
-import { SelectStyled } from './style'
-import { SelectProps } from './types'
-
-const Select = ({
-  children,
-  error = false,
-  fontSize = 1,
-  expand = false,
-  responsible = false,
-  theme = 'light',
-  ...props
-}: SelectProps) => {
+import React, { forwardRef, useImperativeHandle, useRef } from 'react'
+
+import {
+  SelectContainer,
+  SelectCustom,
+  Label,
+  IconLeft,
+  IconRight,
+} from './style'
+import { SelectProps, SelectRef } from './types'
+
+/* eslint-disable */
```

Comment:
Why this eslint-disable.

Reply:
It is necessary, delete the line and you will understand